src/components/common/Button.jsx

```diff
@@ -1,59 +1,31 @@
-import { cva } from 'class-variance-authority';
+import PropTypes from 'prop-types';
 import { forwardRef } from 'react';
 import { motion } from 'framer-motion';
-import { twMerge } from 'tailwind-merge';
+import { getButtonClassName } from './buttonUtils';
```

Comment:
Non-blocking comment:

I do like this approach to keep everything in its corresponding file. However, let's use a [Barrel Pattern](https://medium.com/@denisultanoglu/using-barrel-pattern-in-react-typescript-projects-e8e855730182) to do it. 

I'll create a new small issue to address it in a separate PR.

Reply:
https://github.com/parkiusource/parking-radar-frontend/issues/30